docs/storage.md

```diff
@@ -64,6 +64,33 @@ const server = Server({
 server.run(8000);
 ```
 
+### Flatfile database with [node-persist](https://github.com/simonlast/node-persist)
+
+First, install the necessary packages:
+
+```
+npm install --save node-persist
+```
+
+Then modify your server spec to indicate that you want to connect to Firebase:
```

Comment:
"want to use Flatfile"

Reply:
Already fixed, https://github.com/nicolodavis/boardgame.io/pull/372/commits/d9f4b8e505707da74a3701e73423344f6d7c469f